common/config/rush/shrinkwrap.yaml

```diff
@@ -940,6 +945,197 @@ packages:
     dev: false
     resolution:
       integrity: sha1-LrHQCl5Ow/pYx2r94S4YK2bcXBw=
+  /@uifabric/azure-themes/0.1.0:
+    dependencies:
+      '@uifabric/merge-styles': 6.15.0
+      '@uifabric/set-version': 1.1.3
+      office-ui-fabric-react: 6.128.0
+      tslib: 1.9.3
+    dev: false
+    resolution:
+      integrity: sha512-VlxDWohOV5S5tZ98oV/7dKwnfYgqOey2swKNrEhQG3Cuh4f+4trH+2ahhHdQEX7LuSOz2AWSvWJVgnub4jfHGA==
+  /@uifabric/charting/0.28.5:
+    dependencies:
+      '@microsoft/load-themed-styles': 1.8.54
+      '@types/d3-array': 1.2.1
+      '@types/d3-axis': 1.0.10
+      '@types/d3-scale': 2.0.0
+      '@types/d3-selection': 1.3.0
+      '@types/d3-shape': 1.2.8
+      '@types/d3-time-format': 2.1.0
+      '@uifabric/icons': 6.3.0
+      '@uifabric/set-version': 1.1.3
+      d3-array: 1.2.1
+      d3-axis: 1.0.8
+      d3-scale: 2.0.0
+      d3-selection: 1.3.0
+      d3-shape: 1.2.2
+      d3-time-format: 2.1.3
+      office-ui-fabric-react: 6.128.0
+      prop-types: 15.6.2
+      tslib: 1.9.3
+    dev: false
+    peerDependencies:
+      react: ^0.14.9 || ^15.0.1-0 || ^16.0.0-0
+      react-dom: ^0.14.9 || ^15.0.1-0 || ^16.0.0-0
+    resolution:
+      integrity: sha512-iFwuoFUQoS6H6paxrRsVCxQnw2SqnxhS+EBh54Gb3A6HxS2IKzGAE5c4tnZcqjXNPxXTvXgt5iQg3xycIa95cg==
+  /@uifabric/dashboard/0.43.5:
```

Comment:
I removed the @uifabric/dashboard package, yet after re-running `rush update`, this is getting added.

@pgonzal any ideas here?

Reply:
Is this a blocker for merging the PR?

Reply:
I was hoping to get an answer before merging. I am not sure if this will fix the customer's issue, but I really would prefer not to remove their code.

Reply:
I think to completely clear these out, you'd have to do a `rush update --full` - it is one of the mysteries I don't understand.